### PR TITLE
Debug "Edit Work Order" page hanging on load

### LINKED ISSUE #104: "Edit Work Order" page hangs on loading
Screen Shot 2019-06-13 at 8 36 57 AM

### PR DESCRIPTION
**Pull request by mddilley**
Closes #104

### CODE REVIEW

**Comment by mddilley** [comment]
Added ternary operators to getSupportTechnicianDefaultValue() within AssignTechnicianFields.js and getTaskOrderValues() within TaskOrderField.js. These new operators assign ' ' to the return values of these functions in the case of the Knack API call returning no records. 

The bug was caused by a map function called on the empty string returned from Knack in this case. Records [5c5c8d6bffdb0f62c750f1e1](https://react-knack.netlify.com/work-order/edit/5c5c8d6bffdb0f62c750f1e1) and [5c5c8d6bffdb0f62c750f1e9](https://react-knack.netlify.com/work-order/edit/5c5c8d6bffdb0f62c750f1e9) also reproduced this bug and now load on edit.

File: src/components/WorkOrder/AssignTechnicianFields.js [hunk 1]
```diff
@@ -36,17 +36,21 @@ export default class AssignTechnicianFields extends Component {
     let rawData = this.props.data[`${FIELDS.SUPPORT_TECHNICIANS}_raw`];
 
     let currentData = rawData ? rawData : data;
-
-    return currentData.map(item => ({
-      value: item.id,
-      label: item.identifier
-    }));
+    // handle no assigned Technicians
```
Comment (mateoclarke) [inline src/components/WorkOrder/AssignTechnicianFields.js]: 👏

File: src/components/WorkOrder/Edit.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ class Edit extends Component {
       this.state.rawData,
       updatedFormData
     );
-
+    debugger;
```
Comment (mateoclarke) [inline src/components/WorkOrder/Edit.js]: we should remove this de-🐛-er

Comment (mddilley) [reply src/components/WorkOrder/Edit.js]: Debugger removed. 👍

**Pull request merged**